src/Instance/ItemsList/ItemsListContainer.js

```diff
@@ -3,20 +3,24 @@ import React, {
   useState,
 } from 'react';
 import PropTypes from 'prop-types';
+import { isEmpty } from 'lodash';
 
 import { stripesConnect } from '@folio/stripes/core';
 
 import ItemsList from './ItemsList';
 
-const ItemsListContainer = ({ holding, mutator, ...rest }) => {
+const ItemsListContainer = ({ holding, mutator, setOpen, ...rest }) => {
```

Comment:
it's not obvious from his component what `setOpen`  is for

Reply:
`setOpen` function controls the expanding/collapsing of the accordion depending on the presence/absence of elements in it.